frontend/src/locales/en.json

```diff
@@ -357,6 +357,21 @@
     "shareMyLink": "Share my link",
     "reportBug": "Report Bug"
   },
+  "notices": {
+    "betaWarning": {
+      "heading": "As we're currently in beta, we do have some issues we're dealing with:",
```

Comment:
Da wir aktuell in der Beta-Phase sind, hier einige Tips um Probleme zu umgehen:

(As we're currently in beta, here are a few tips to avoid problems) - I think this sounds a bit more positive than "issues"